### ISSUE TITLE
A11y: poor color contrast for progress bar /w Antarctic

### ISSUE THREAD

**Issue by shield-9** [weseek/growi#1389]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        | Ubuntu 18.04.3|
|GROWI     |3.5.17 and 3.6.0.-RC|
|node.js   |10.17.0|
|npm       |x6.11.3|
|Using Docker|yes (Kubernetes) |
|Using [growi-docker-compose][growi-docker-compose]|no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        |Windows 10 1909 18363.476|
|browser   |Chrome 78.0.3904.108|

How to reproduce? (再現手順)
---------------------------

1. Set theme to "antarctic"
1. Open *Full Text Search management*

What happens? (症状)
---------------------

It is not easy to recognize the status from progress bar due to poor color contrast.
![Full Text Search management - GroupWiki](https://user-images.githubusercontent.com/3660715/69506153-b1163d80-0f70-11ea-8bd1-aaa9d49855c3.png)

What is the expected result? (期待される動作)
-------------------------------------------

All components should have sufficient color contrast.
